corehq/apps/app_manager/tests/data/list_apps/list_apps.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <apps>
-  <app profile="http://onering.com/a/rohan/apps/download/app_id/profile.xml?latest=true" domain="rohan" name="Untitled Application" environment="production" media-profile="http://onering.com/a/rohan/apps/download/app_id/media_profile.xml?latest=true"/>
```

Comment:
hey, the way I set this up, it always returns only latest starred versions. 
I'm not actually sure why the previous url contains `latest=true`, as I don't think that parameter does anything. The thing that gets the "version" in the profile url is confusingly the `app_id` (which here is just mocked out as `app_id`).

At a later date, we can definitely let the user select which build they want, and we can discuss how we want that to happen.